javascript/src/index.ts

```diff
@@ -456,6 +457,25 @@ class Endpoint {
       { appId, endpointId, eventExampleIn, ...options }
     );
   }
+
+  public updateOauthConfig(
```

Comment:
```suggestion
  public oauthConfigUpdate(
```
And same for the other ones.